### ISSUE TITLE
about loss function

### ISSUE THREAD

**Issue by zqq-judy** [Hmslab/Adapose#2]
Hello! 
1、where is the mask loss in the code？

2、`self.loss_simp, dist, idx, dist2 = self._get_simplification_loss(self.simplified_points,
                                                                         self.pcloud,
                                                                         self.simplified_num)` 
Is this line of code used to calculate the distance loss? What is the meaning of `self.loss_simp` ?

3、Are the parameters in the code different from those in the paper?
              code： λ-3d=1， λ-2d=0.01, λ-con=0.0001
              paper：λ-3d=10， λ-2d=0.1, λ-con=0.001
    What about λ-sample?

Thanks.

**Comment by Hmslab** [comment]
Thanks for you comments !
1. We missed the mask loss term when sorting out the code, and this will be added in  a subsequent version update. 
2.  The self.loss_simp is the "Distance Loss" described in our paper,  we encourage the generated sampling centers R to be close
to the original point clouds P
3.  We use  λ-3d=1， λ-2d=0.01, λ-con=0.0001,  and the λ-sample = 1

**Comment by zqq-judy** [comment]
Hello!

1. Can you share the test or training code on the NTU RGB-D dataset if convenient?
2. The NTU RGB-D dataset looks like it just has 3D pose, how do you get its 2D pose?

Best wishes!

**Comment by Hmslab** [comment]
1. The training and testing code on the NTU RGB-D dataset is similar to ITOP,  except that the number of joints and the camera internal parameters.  
2.  The NTU RGB-D dataset contains GT 2D poses.

**Comment by zqq-judy** [comment]
1、how to get skeleton in flags.py file?
`skeleton = [0.18099216, 0.30136532, 0.3083691, 0.1809909, 0.30517647, 0.3111792, 0.26156965, 0.4784025, 0.46032718, 0.26156482, 0.47747827, 0.4607155, 0.23529877, 0.23755938]`
I want to get this skeleton for the NTU dataset.

2、And  `is_valid` ，`visible_joints`  about NTU?

**Comment by Hmslab** [comment]
1. The "skeleton" in the flags.py file does not represent a specific skeleton. The file to get the NTU skeletons and poses we have updated in the NTU folder. 
2. is_valid and visible_joints are about ITOP

**Comment by zqq-judy** [comment]
ok, Thanks!
